docs/api.html

```diff
@@ -87,7 +88,22 @@
 	Those who want to implement custom connectors can see the <a href="/{{version}}/javadoc/index.html?org/apache/kafka/connect" title="Kafka 0.10.2 Javadoc">javadoc</a>.
 	<p>
 
-	<h3><a id="legacyapis" href="#streamsapi">2.5 Legacy APIs</a></h3>
+	<h3><a id="adminapi" href="#adminapi">2.5 AdminClient API</a></h3>
+
+	The AdminClient API supports managing and inspecting topics, brokers, acls, and other Kafka objects.
+	<p>
+	To use the AdminClient API, add the following Maven dependency:
+	<pre>
+		&lt;dependency&gt;
+			&lt;groupId&gt;org.apache.kafka&lt;/groupId&gt;
+			&lt;artifactId&gt;kafka-clients&lt;/artifactId&gt;
+			&lt;version&gt;0.10.2.0&lt;/version&gt;
+		&lt;/dependency&gt;
+	</pre>
+	For more information about the AdminClient APIs, see the <a href="/{{version}}/javadoc/index.html?org/apache/kafka/clients/AdminClient.html" title="Kafka 0.10.2 Javadoc">javadoc</a>.
```

Comment:
Can we use `{{version}}`` in the title as well? If possible, it seems like we should do that for all 5 APIs.